pkg/crc/ssh/ssh.go

```diff
@@ -20,9 +22,29 @@ func CreateRunnerWithPrivateKey(driver drivers.Driver, privateKey string) *SSHRu
 
 // Create a host using the driver's config
 func (runner *SSHRunner) Run(command string) (string, error) {
-	return drivers.RunSSHCommandFromDriver(runner.driver, runner.privateSSHKey, command)
+	return runSSHCommandFromDriver(runner.driver, runner.privateSSHKey, command)
 }
 
 func (runner *SSHRunner) SetPrivateKeyPath(path string) {
 	runner.privateSSHKey = path
 }
+
+func runSSHCommandFromDriver(d drivers.Driver, sshPrivateKeyPath, command string) (string, error) {
```

Comment:
`func (runner *SSHRunner) runSSHCommandFromDriver(command string) (string, error) {}`

Reply:
@cfergeau why do you want this function to part of `SSHRunner` struct? I thought this is something we are not exposing as part of `CreateRunner` or `CreateRunnerWithPrivateKey` invocation?

Reply:
The main benefit is that we don't need to pass sshPrivateKey and drivers.Driver separately, they are only known.